aiohttp/web_ws.py

```diff
@@ -17,6 +18,9 @@
 
 THRESHOLD_CONNLOST_ACCESS = 5
 
+WebSocketReady = namedtuple('WebSocketReady', 'ok protocol')
+WebSocketReady.__bool__ = lambda self: self.ok
```

Comment:
I believe this code is not idiomatic.
Please use inheritance as python doc suggests in https://docs.python.org/3/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields
```
>>> class Point(namedtuple('Point', 'x y')):
...     __slots__ = ()
...     @property
...     def hypot(self):
...         return (self.x ** 2 + self.y ** 2) ** 0.5
...     def __str__(self):
...         return 'Point: x=%6.3f  y=%6.3f  hypot=%6.3f' % (self.x, self.y, self.hypot)
```